Misc/NEWS.d/next/Build/2024-06-11-00-38-05.gh-issue-120326.JHSDF1.rst

```diff
@@ -0,0 +1,2 @@
+Fix build error on Windows when ``--disable-gil`` and ``--experimental-jit``
+options are combined.
```

Comment:
I prefer to mention the "domain" first, so the reader can quickly skip if it's unrelated to their use cases:

```suggestion
On Windows, fix build error when ``--disable-gil`` and ``--experimental-jit``
options are combined.
```